package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "bookshelf_app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@chakra-ui/react": "^2.8.2",
+    "@emotion/react": "^11.13.0",
+    "@emotion/styled": "^11.13.0",
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
```

Comment:
Can we move these to be `devDependencies` ?

Reply:
Also `eslint` and `prettier`

Reply:
Remove eslint and prettier?

Reply:
Can move them to `devDependencies` I mean

Reply:
thanks!